src/login.rs

```diff
@@ -248,9 +249,58 @@ impl<S: TokenStorage> LoginCredentials for RefreshingLoginCredentials<S> {
                 .map_err(RefreshingLoginError::UpdateError)?;
         }
 
+        let mut current_login = self.user_login.lock().await;
+
+        let login = match &*current_login {
+            Some(login) => login.clone(),
+            None => {
+                let response = self
+                    .http_client
+                    .get("https://api.twitch.tv/helix/users")
+                    .header("Client-Id", &self.client_id)
+                    .bearer_auth(&current_token.access_token)
+                    .send()
+                    .await
+                    .map_err(RefreshingLoginError::RefreshError)?;
+
+                log::info!("Fetching username: {}", response.status());
```

Comment:
I don't like this log message. There's not much information in `Fetching username: OK` in my opinion. I'd rather have it removed.

```suggestion
```

Reply:
My bad, this was left in after debugging.